genesyscloud/architect_flow/resource_genesyscloud_architect_flow_proxy.go

```diff
@@ -185,3 +297,124 @@ func getAllArchitectFlowsFn(ctx context.Context, p *architectFlowProxy, name str
 
 	return &totalFlows, nil, nil
 }
+
+func generateDownloadUrlFn(a *architectFlowProxy, flowId string) (string, error) {
+	log.Printf("Creating export job for flow %s", flowId)
+	jobId, resp, err := a.createExportJob(flowId)
+	if err != nil {
+		log.Println(err)
```

Comment:
Should we add some more context to this log message like "Error in generateDownLoadUrlFN.  Otherwise the error message might not give an indication what is wrong.